mdbenchmark/analyze.py

```diff
@@ -110,7 +110,11 @@ def analyze(directory, plot, ncores, save_csv):
         ax = fig.add_subplot(111)
 
         df = pd.read_csv(save_csv)
-        ax = plot_over_group(df, plot_cores=False, ax=ax)
+        if ncores:
+            console.warn(
+                "Ignoring any values you specified and parsing number of cores from log files."
```

Comment:
shouldn't this be the other way around?
```if not ncores:```
If the user gives a value for ncores the value overrides the automated parsing.

Reply:
That is what it did before. The current implementation (re-write for `mdbenchmark plot`), only accepts `--plot-cores` as an option and then plots the number of cores, instead of nodes.

This does lead to the problem: if we cannot extract number of cores values from certain log files, but the performance is there, then we will not plot these values.